app/src/main/java/com/hyperion/ths/marvel_03/ui/main/MainViewPageAdapter.java

```diff
@@ -0,0 +1,59 @@
+package com.hyperion.ths.marvel_03.ui.main;
+
+import android.content.Context;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentStatePagerAdapter;
+import android.support.v4.view.PagerTabStrip;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+import com.hyperion.ths.marvel_03.R;
+import com.hyperion.ths.marvel_03.ui.favorite.FavoriteFragment;
+import com.hyperion.ths.marvel_03.ui.hero.HeroFragment;
+
+/**
+ * Created by ths on 27/05/2017.
+ */
+
+public class MainViewPageAdapter extends FragmentStatePagerAdapter {
+    private Context mContext;
+    private String[] mTabsTitle = { "Hero", "Favorite" };
+    private int[] mTabsIcons = { R.drawable.ic_hero_selector, R.drawable.ic_favorite_selector };
+
+    public MainViewPageAdapter(FragmentManager fm, Context context) {
+        super(fm);
+        mContext = context;
+    }
+
+    public View getTabView(int position) {
+        View view = LayoutInflater.from(mContext).inflate(R.layout.custom_tablayout,null);
```

Comment:
WARNING: [InflateParams] Avoid passing `null` as the view root (needed to resolve layout parameters on the inflated layout's root element)
 When inflating a layout, avoid passing in null as the parent view, since otherwise any layout parameters on the root of the inflated layout will be ignored.